android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -139,6 +139,9 @@ public class RNCWebViewManager extends SimpleViewManager<WebView> {
   public static final int COMMAND_LOAD_URL = 7;
   public static final int COMMAND_FOCUS = 8;
 
+  // commands added by Discord
+  public static final int COMMAND_RELEASE = 4001;
```

Comment:
I'm starting this from 4001, so there will be fewer collisions with other commands in the future. Later, I may make a PR for these changes into the non-discord react-native-webview repo. If I'll do that, I'll change this value to `9`.